src/inspector/inspector_socket_server.cpp

```diff
@@ -338,10 +338,18 @@ bool InspectorSocketServer::HasTargets() {
 
 bool InspectorSocketServer::Start() {
   state_ = ServerState::kRunning;

```

Comment:
>state_ = ServerState::kRunning; [](http://example.com/codeflow?start=2&length=31)

Since on the Stop() operation we reset the delegate, I assume that we must not be able to Start the server after it was stopped. If it is correct, then we must check the state_ to ensure that we cannot change it to kRunning after it is already kStopped.

Reply:
Yes, that cannot happen; added a CHECK.